src/QMCWaveFunctions/tests/test_TrialWaveFunction_He.cpp

```diff
@@ -192,8 +192,7 @@ UPtrVector<ParticleSet::ParticleGradient_t> create_particle_gradient(int nelec,
   UPtrVector<ParticleSet::ParticleGradient_t> G_list;
   for (int i = 0; i < nentry; i++)
   {
-    G_list.push_back(std::make_unique<ParticleSet::ParticleGradient_t>());
-    G_list.back()->resize(nelec);
+    G_list.emplace_back(std::make_unique<ParticleSet::ParticleGradient_t>(nelec));
```

Comment:
Sorry I forgot to mention that you can remove the surrounding `{}` as well.

Reply:
It is a tedious request. So far I found clang-tidy can do it but not clang-format.